### DISCUSSION TITLE
How to check if unit already exists?

### DISCUSSION THREAD

**Discussion by petermarks12** [josdejong/mathjs#2659] [Q&A] [answered]
Hi,

I have created a custom unit using math.createUnit().  Now, I want to check if that unit exist or not, so that I should not end up creating same unit again.

Is there any way to retrieve list of all units, so that I can verify this?

**Comment by josdejong** [comment, marked as answer]
Hm, you could use `math.type.Unit.isValuelessUnit('cm')` I think.
